### PR TITLE
Npad redirect

### PR DESCRIPTION
**Pull request by fernandalavalle**
The way mlab-ns handles npad?format=redirect http requests is currently broken. If a request does not specify a port, the request is answered with a 404. This short term solution specifies port  8000 for all npad redirect requests. This is a constant port that should not need to be changed.

---
This change is [Reviewable](https://reviewable.io/reviews/m-lab/mlab-ns/92)

### CODE REVIEW

**Comment by fernandalavalle** [comment]
Addresses: 
https://github.com/m-lab/mlab-ns/issues/38

File: server/mlabns/handlers/lookup.py [hunk 1]
```diff
@@ -23,7 +23,7 @@ def _create_tool_url(fqdn, http_port):
     Returns:
         A constructed tool HTTP URL with protocol and http port.
     """
-    return 'http://%s:%s' % (fqdn, http_port)
+    return str('http://%s:%s' % (fqdn, http_port))
```
Comment (pboothe) [inline server/mlabns/handlers/lookup.py]: This change looks redundant and like it does nothing?  Did you mean:

`return 'http://%s:%s' % (fqdn, str(http_port))`

or maybe

`return 'http://%s:%d' % (fqdn, http_port)`

?

Comment (fernandalavalle) [inline server/mlabns/handlers/lookup.py]: Yeah, just looking at it I agree it looks redundant. If I take it out though, I get the unicode error we see on mlab-ns now (http://mlab-ns.appspot.com/npad?format=redirect). So I definitely have to cast something as a string, but I'm open to more clever/logical approaches.

Comment (pboothe) [inline server/mlabns/handlers/lookup.py]: Put a comment above it saying
  `# Use str() to ensure we return a non-unicode string` 
and I am happy.

**Comment by pboothe** [comment]
LGTM once the comment is added.

**Pull request merged**